src/test/java/com/commercetools/sync/customers/utils/StoreUpdateActionUtilsTest.java

```diff
@@ -0,0 +1,323 @@
+package com.commercetools.sync.customers.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.customers.CustomerDraft;
+import io.sphere.sdk.customers.CustomerDraftBuilder;
+import io.sphere.sdk.customers.commands.updateactions.AddStore;
+import io.sphere.sdk.customers.commands.updateactions.RemoveStore;
+import io.sphere.sdk.customers.commands.updateactions.SetStores;
+import io.sphere.sdk.models.KeyReference;
+import io.sphere.sdk.models.ResourceIdentifier;
+import io.sphere.sdk.stores.Store;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
```

Comment:
```suggestion
import java.util.List;
import org.junit.jupiter.api.Test;
```

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/591/commits/fcc602b45b040acfa647b576c7665901ba2ec374

Reply:
Are you using a different mechanism to order those, those import orders do not look important for reviews maybe share your ide settings? Normally an import is added automatically by ide, I never wrote import by hand.